### ISSUE TITLE
is there any way to edit how the culling behaves?

### ISSUE THREAD

**Issue by Entityexist** [AdrienTD/c47edit#12]
Hello
i unexpectetly found your project of making c47 edit and its like a miracle for me cause im working on RTX Remix Remaster Project
and im in a bit of pickle cause culling in glacier engine causes me a headache and i would love to know if there is a way to edit or disable it by any chance and not only for C47 but for hitman 2 that appears to work on the same engine

by the way i love your project and work 

cant wait to any respond 

Best Regards Entity

**Comment by AdrienTD** [comment]
Hi,
By culling I assume you mean the drawing distance?

If you want to increase the draw distance, I suggest looking at the [Hitman Unlocker](https://github.com/OrfeasZ/HitmanUnlocker) project which lets you configure the draw distance during gameplay by installing a single file.

If it does not work, you can also use c47edit to change the draw distance: simply open a valid scene zip file, then go to **Tools > ZDefines**. There you can change the camera's near/far distance as well as the FOV. However doing so would only apply to the scene where you changed the values, so you would have to change the values of every single scene, which can take time.

**Comment by Entityexist** [comment]
culling i would like to render whole game plus all objects that are behind the camera view

**Comment by AdrienTD** [comment]
Yes, in that case use the Hitman Unlocker, or with c47edit change the Camera Far Distance in the ZDEFINES to make more objects appear at far distance. However, be careful that putting a very large far distance can decrease performance, especially in large areas like Columbia.

**Comment by Entityexist** [comment]
what about the frustrum of the camera can i force to get more objects loaded from behind camera?

**Comment by AdrienTD** [comment]
Honestly I don't know. Maybe you could change the near distance to a negative value, but I think that would break the game?

There is also a cheat code to enable rendering rooms when they are covered ( [key Y after enabling debug mode](https://hitman.fandom.com/wiki/Hitman:_Codename_47/Cheats) ), but not sure if that will also render rooms behind the camera.

**Comment by Entityexist** [comment]
it will shutdown ZGates and game will render everything whats in frustrum of camera

you might be only single person i can ask questions about this game 

don't worry if you cant answer my questions

i found that some fof the code is connected to ZGEOM and FOV and Len

![image](https://github.com/user-attachments/assets/779da0d8-e025-4a4f-8c8c-cc32180e3e6f)

im looking for help cause im trying to work on hitman projects with path traced lighning via RTX Remix Runtime

and there is big issue with light leaking into culled scene ;D

i have this Variables is there any way to push them to the game and change their values?
